UPGRADE-3.0.md

```diff
@@ -0,0 +1,15 @@
+# Upgrade from 2.x to 3.0
+
+- The signature of the `captureMessage()` global function changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `captureException()` global function changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `captureEvent()` global function changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `captureLastError()` global function changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `ClientInterface::captureMessage()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `ClientInterface::captureException()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `ClientInterface::captureEvent()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `ClientInterface::captureLastError()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `HubInterface::captureMessage()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `HubInterface::captureException()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `HubInterface::captureEvent()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `HubInterface::captureLastError()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
+- The signature of the `Event::getId()` method changed to return an instance of the `Sentry\EventId` class instead of a `string`
```

Comment:
Can we make this less verbose? Something like: "The following functions now return `EventId` instead of `string`

Reply:
Might not be a bad plan to summarise all functions that changed to be explicit.